### ISSUE TITLE
AMD, CommonJS and Plain Browser Support

### ISSUE THREAD

**Issue by se-fDelpech** [Kinvey/angular-sdk#5]
Hi,
Would that be possible to make the library available for any sources? Right now it's a plain global definition that does not fit well with webpack.
In reference to https://gist.github.com/pazguille/3923164 , the export of the library as a module can be done to get AMD, commonJs and plain browser support.

**Comment by ThomasConner** [comment]
@se-fDelpech Can you share your webpack config? You should already be able to use the SDK with a webpack built app.

**Comment by se-fDelpech** [comment]
Hi,
It was maybe more related to the beta build issue, but the idea would be to use it as a module :

```
var kinvey = require('kinvey-angular-sdk');

angular.module('myapp', [
  kinvey.name,
  ...
])
```

But to make it work right now, I have to use: 

```
require('kinvey-angular-sdk');

angular.module('myapp', [
  'kinvey',
  ...
])
```

It's working, it's just that it does not align with the standard way to import module.

**Comment by ThomasConner** [comment]
Ok good point. We always encourage pull requests if there is something you see that needs to be fixed or could be improved.

If you would like to make a pull request with the fix let us know otherwise we will make it so the SDK exports a module with a name property set to `kinvey`.

If you are looking to make a pull request with the fix, take a look at [src/index.js](https://github.com/Kinvey/angular-sdk/blob/master/src/index.js). At the bottom of the file you would need to export an object with a name property set to `kinvey`.

**Comment by ThomasConner** [comment]
I didn't mean to close the issue.

**Comment by se-fDelpech** [comment]
Yes sure, I can do that.

**Comment by se-fDelpech** [comment]
Should I fork from the master or the dev?

**Comment by ThomasConner** [comment]
This issue has been fixed with the release of version 3.0.1 of the [kinvey-angular-sdk](https://www.npmjs.com/package/kinvey-angular-sdk) package.
